lib/webdriveragent.js

```diff
@@ -286,6 +300,20 @@ class WebDriverAgent {
       return await this.getStatus();
     }
 
+    if (this.isRealDevice && this.usePreinstalledWDA) {
```

Comment:
so this only works on real devices?

Reply:
Yes, so far. I guess we could do something for simulators as well like idb does(?). I haven't checked tho.

For simulator, probably `-xctestrun` option (available via `useXctestrunFile`) is the most reasonable for prebuild wda

Reply:
I would also describe the process for simulator in a separate doc if possible. This is a major improvement, which allows to save lot of time on session startup

Reply:
Let me write something after https://github.com/appium/appium-xcuitest-driver/pull/1609
Probably https://github.com/appium/ruby_lib_core/blob/master/test/test_helper.rb#L217-L241 helps. In the past, some xcode version? it did not work, but I saw recent Xcode 14 worked with `appium:bootstrapPath` and `appium:useXctestrunFile`. For simulator. maybe this xcodebuild is good